VideoLocker/res/color/nav_btn_selector.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_enabled="true" android:color="@color/edx_brand_primary_accent" />
+    <item android:color="@color/edx_grayscale_neutral_base" />
```

Comment:
Does Android have a default disabled styling it applies? If not, seems fine as a color. Might be worth naming it as disabled_button or something, so we use it everywhere?